include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -146,22 +146,6 @@ class TTIR_ElementwiseOp<string mnemonic, list<Trait> traits = []> :
     let results = (outs Variadic<AnyRankedTensor>:$results);
 }
 
-class TTIR_ElementwiseUnaryOp<string mnemonic, list<Trait> traits = []> :
-    TTIR_ElementwiseOp<mnemonic, traits> {
-    let summary = "Eltwise unary op.";
-    let description = [{
-      Eltwise unary op.
-    }];
-
-    let builders =
-    [
-      OpBuilder<(ins "Value": $in, "Value": $out, "ArrayAttr": $operand_constraints),
-      [{
-        build($_builder, $_state, {out.getType()}, in, out, operand_constraints);
-      }]>
-    ];
-}
-
```

Comment:
Did this need to move?  I think organizationally it makes more sense to keep it here next to `TTIR_ElementwiseBinaryOp`

Reply:
I think I slightly prefer to keep the generic stuff colocated and the named ops colocated, this is how I would organize it if I were to break it into separate files.

```
# Named ops section
TTIR_ElementwiseOp
TTIR_ElementwiseUnaryOp : TTIR_ElementwiseOp
TTIR_ElementwiseBinaryOp : TTIR_ElementwiseOp
TTIR_ReductionOp

# Generic ops section
TTIR_GenericElementwiseUnaryOp : TTIR_ElementwiseUnaryOp
TTIR_GenericElementwiseBinaryOp : TTIR_ElementwiseBinaryOp
# Generic Reduction ops
...
```

Reply:
If I understood properly, you suggest:

```mlir
TTIR_ElementwiseOp
TTIR_ElementwiseUnaryOp
# ops derived from TTIR_ElementwiseUnaryOp
TTIR_ElementwiseBinaryOp
# ops derived from TTIR_ElementwiseBinaryOp
TTIR_ReductionOp
# ops derived from TTIR_ReductionOp
TTIR_EmbeddingOp
... # and other ops derived from TTIR_DPSOp

TTIR_GenericElementwiseUnaryOp
# ops derived from TTIR_GenericElementwiseUnaryOp
TTIR_GenericElementwiseBinaryOp
# ops derived from TTIR_GenericElementwiseBinaryOp
```

is that right?